<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget_Test">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>211</height>
    </rect>
   </property>
   <property name="gridVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>381</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_prevMonth">
      <property name="text">
       <string>Prev. Month</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_today">
      <property name="text">
       <string>Today</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_nextMonth">
      <property name="text">
       <string>Next Month</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="lbl_selectedDate">
      <property name="text">
       <string>Selected Date Here</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_currentPage">
      <property name="text">
       <string>Current Page Here</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
